vernac/metasyntax.mli

```diff
@@ -63,3 +63,5 @@ val pr_custom_grammar : string -> Pp.t
 val with_syntax_protection : ('a -> 'b) -> 'a -> 'b
 
 val declare_custom_entry : locality_flag -> string -> unit
+
+val pr_level : Constrexpr.notation -> Notation.level -> Extend.constr_entry_key list -> Pp.t
```

Comment:
Maybe a comment here would be warranted?

Reply:
Will do